### PR TITLE
Address GKE operator connectivity issue

### LINKED ISSUE #205: Document workaround for operator in GKE
https://discord.com/channels/933706914808889356/1187405763338064003

### PR DESCRIPTION
**Pull request by infiniteregrets**
closes #205

### CODE REVIEW

File: content/docs/overview/faq.md [hunk 1]
```diff
@@ -199,3 +199,7 @@ in the agent configuration:
 }
 ```
 
+### `mirrord operator status` fails with `503 Service Unavailable` on GKE
```
Comment (eyalb181) [inline content/docs/overview/faq.md:202]: This is a mirrord for Teams issue, so not relevant to this FAQ. Please create an FAQ page under the `mirrord for Teams` section, and put this question under a Common Issues section.

File: content/docs/overview/faq.md [hunk 2]
```diff
@@ -199,3 +199,7 @@ in the agent configuration:
...
+
+If private networking enabled, it is likely due to firewall rules blocking the mirrord operator's API service from the API server. To fix this, add a firewall rule that allows your cluster's master nodes to access TCP port 3000 in your cluster's pods. Please refer to the [GCP docs](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules) for information.
```
Comment (eyalb181) [inline content/docs/overview/faq.md:204]:
```suggestion
If private networking is enabled, it is likely due to firewall rules blocking the mirrord operator's API service from the API server. To fix this, add a firewall rule that allows your cluster's master nodes to access TCP port 3000 in your cluster's pods. Please refer to the [GCP docs](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules) for information.
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/docs/overview/faq.md (hunk 1 above)
Comment (infiniteregrets) [reply content/docs/overview/faq.md]: Is "General" more suited here instead of "Common issues"?

**Comment by infiniteregrets** [comment]
Accidentally commited same js twice one in the HTML file and other in alert.js, so removing that

File: layouts/partials/header/alert.html [hunk 3]
```diff
@@ -4,19 +4,6 @@
   {{ .Site.Params.alertText | safeHTML }}
   <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close" onclick="dismissAlert()"></button>
 </div>
-
-<script>
```
Comment (eyalb181) [inline layouts/partials/header/alert.html:8]: ?

File: content/docs/teams/FAQ/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+title: "FAQ"
+description: "FAQ for mirrord for Teams"
+draft: false
+images: []
+linktitle: "FAQ"
+weight: 130
+toc: true
+---
+
+## General
```
Comment (eyalb181) [inline content/docs/teams/FAQ/index.md:11]:
```suggestion
## Common Issues
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: layouts/partials/header/alert.html (hunk 3 above)
Comment (infiniteregrets) [reply layouts/partials/header/alert.html:8]: I already commented here:

> Accidentally commited same js twice one in the HTML file and other in alert.js, so removing that here

File: content/docs/overview/faq.md (hunk 1 above)
Comment (eyalb181) [reply content/docs/overview/faq.md]: Just noticed you commented here. I don't think so, it's a very specific issue.

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**